packages/twenty-server/src/modules/workflow/workflow-executor/workflow-actions/types/workflow-action-settings.type.ts

```diff
@@ -0,0 +1,23 @@
+import { WorkflowCodeActionSettings } from 'src/modules/workflow/workflow-executor/workflow-actions/code/types/workflow-code-action-settings.type';
+import { WorkflowSendEmailActionSettings } from 'src/modules/workflow/workflow-executor/workflow-actions/mail-sender/types/workflow-send-email-action-settings.type';
+import { WorkflowRecordCRUDActionSettings } from 'src/modules/workflow/workflow-executor/workflow-actions/record-crud/types/workflow-record-crud-action-settings.type';
+
+export type OutputSchema = object;
+
+export type BaseWorkflowActionSettings = {
+  input: object;
+  outputSchema: OutputSchema;
+  errorHandlingOptions: {
+    retryOnFailure: {
```

Comment:
seems both bool are mutually exclusive, right? 
If so, should we create a strategy enum instead? @thomtrp 
If both are true, does that mean it will retry once and continue if still fails? If so you can ignore my comment. You could take example on queue system libraries such as bullmq, usually we want to provide the number of retry etc.
Up to you!

Reply:
> If both are true, does that mean it will retry once and continue if still fails?
Yes, it will retry `MAX_RETRIES_ON_FAILURE` which is a repo constant for now, and then continue if it fails